js/views/transactionsVw.js

```diff
@@ -281,6 +285,118 @@ module.exports = baseVw.extend({
     tabWrapper.append(orderShort.render().el);
   },
 
+  clickDownloadCSV: function(e){
+    var targBtn = $(e.target);
+    targBtn.closest('.btn').addClass('loading');
+    this.downloadCSV(targBtn);
+  },
+
+  downloadCSV: function(targBtn){
+    var self = this,
+        rawData = [],
+        calls = [],
+        dataCSV = '',
+        tempAnchor = document.createElement('a'),
+        dataBlob,
+        exportData = function(data){
+          "use strict";
+          dataCSV = Papa.unparse(data);
```

Comment:
Since `dataCSV` is just used in this function, can it be declared here as opposed to the outer scope?